docs/HowTo/Get-Started/Starting-node.md

```diff
@@ -94,6 +94,20 @@ rpc-http-enabled=true
 data-path="/tmp/tmpdata-path"
 ```
 
+!!! caution
+
+    We do not recommend using the following settings in a production environment:
+    
+    * Enabling the HTTP JSON-RPC service
+      ([`--rpc-http-enabled`](../../Reference/CLI/CLI-Syntax.md#rpc-http-enabled)) and setting
+      [`-rpc-http-host`](../../Reference/CLI/CLI-Syntax.md#rpc-http-host) to 0.0.0.0 exposes the
```

Comment:
exposes -> to expose 

Suggested edit to make parallel with the other two bullets.